src/config/openpgp/index.md

```diff
@@ -0,0 +1,7 @@
+# OpenPGP Tooling
+
+## GnuPG
+
+Void ships both GnuPG legacy (as `gnupg1`) and GnuPG stable (as `gnupg`). A
+package for the LTS branch is currently not available. Details for configuring
```

Comment:
Drop the mention of the LTS package; it doesn't add a lot of useful information.